artifacts/install/operator-configs/controller-conf.yaml

```diff
@@ -48,30 +48,30 @@ data:
   apim-analytics-deployment-readinessProbe-failureThreshold: "3"
 
   # pattern specific configs for apim-instances
-  p1-apim-deployment-image: "wso2/wso2am:3.0.0"
+  p1-apim-deployment-image: "pubudu/wso2am:3.1.0-rc1"
   p1-apim-deployment-resources-requests-cpu: "2000m"
   p1-apim-deployment-resources-requests-memory: "2Gi"
   p1-apim-deployment-resources-limits-cpu: "2000m"
   p1-apim-deployment-resources-limits-memory: "2Gi"
 
   # custom pattern specific configs for apim-instances
-  pX-apim-deployment-image: "wso2/wso2am:3.0.0"
+  pX-apim-deployment-image: "pubudu/wso2am:3.1.0-rc1"
```

Comment:
Why there are user-specific image details?

Reply:
Used only for pre-release testing. Fixed in latest code.